### DISCUSSION TITLE
Remove hover on table rows on hover

### DISCUSSION THREAD

**Discussion by JorgeDamico** [NG-ZORRO/ng-zorro-antd#8069] [General]
I need remove hover on table rows on hover, in nz-table

**Comment by anhnv-dev** [comment]
You can use css
```
.ant-table-tbody > tr.ant-table-row:hover > td,
.ant-table-tbody > tr > td.ant-table-cell-row-hover {
    background: transparent;
}
```

**Reply by XiaLing233** [reply]
This nearly works for me, but a `!important` should be added after `transparent` to enhance the style's priority(I'm using antd-vue for vue3 under a similar situation).
